src/modules/host/host-list/host-list.js

```diff
@@ -0,0 +1,62 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("hostController", hostController);
+
+    /*@ngInject*/
+    function hostController($scope, $interval, utils, config) {
+        var vm = this;
+
+        vm.init = init;
+        vm.setupHostListData = setupHostListData;
+        vm.init();
```

Comment:
There is no need of making init  function available to the scope of controller as it is not going to called from html. Hence replace vm.init to init